models/gc_nnunet_pancreas/config/default.yml

```diff
@@ -0,0 +1,35 @@
+general:
+  version: 1.0
+  data_base_dir: /app/data
+  description: base configuration for GC NNUnet Pancreas model (dicom to dicom)
+
+execute:
+- DicomImporter
+- MhaConverter
+- GCNNUnetPancreasRunner
+- DsegConverter
+- DataOrganizer
+
+modules:
+  DicomImporter:
+    source_dir: input_data
+    import_dir: sorted_data
+    sort_data: true
+    meta: 
+      mod: '%Modality'
+
+  MhaConverter:
```

Comment:
Note, that `targets` has been renamed to `in_datas` some while ago.
Documentation: https://github.com/MHubAI/models/pull/45#pullrequestreview-1808688218

Reply:
I'll update it.